packages/fxa-payments-server/src/components/PlanDetails/index.scss

```diff
@@ -1,22 +1,6 @@
 @import '../../../../fxa-content-server/app/styles/breakpoints';
 
-.payment-error ~ .plan-details-component,
-.payment-confirmation + .plan-details-component {
-  margin: 16px 16px 32px;
-
-  @include max-width('tablet') {
-    grid-row: 3/4;
-  }
-}
```

Comment:
As per our Slack chat, removing this causes two display issues in all subscription success and payment error pages in mobile:

1. there is no space above `Plan Details`, separating it from the success or error containers
2. the top corners are not rounded

Since this only happens for success and error routes, I recommend creating a custom class in `tailwind.css` that makes all the corners round and adds a top margin. E.g.:
```
.plan-details-component-card.custom-class-name {
  @apply rounded-lg mt-6;
}
```
Then, pass `className` as a prop to `PlanDetails` from all affected components/routes.

Additionally, please add `className` to `PlanDetails` as shown below:
```
 43        className = '',
...
172        <div className={`plan-details-component-card ${className}`}> 
```
If you have an alternative way this can be done, I'm down for it! I couldn't figure out a way to easily convert to TW with the above combinators. 😕

Reply:
Thanks for solving this, @sardesam! I was really stumped by it.